project.clj

```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.1")
 (def schema-version "1.1.7")
 
-(defproject threatgrid/flanders "0.1.19-SNAPSHOT"
+(defproject threatgrid/flanders "0.1.19"
```

Comment:
Are you going to release a new version of Flanders with this PR?

PR for fixes or features are squashed and merged to keep only one commit. It's easier to manage if we have to revert a PR.

PR for release are merged to keep all commits

Reply:
We generally create another PR with 2 commits for releases.

Reply:
Makes sense to me.